src/lib/fix-pmc.js

```diff
@@ -0,0 +1,26 @@
+import { options } from 'preact';
+
+const classNameDescriptor = {
```

Comment:
What does this file do?

Reply:
If we still need this, some comments would be handy, just to communicate the intent.